src/navigator/IFrameNavigator.ts

```diff
@@ -2939,8 +2935,6 @@ export default class IFrameNavigator implements Navigator {
       }
       if (isCurrentLoaded) {
         console.log("is currently loaded");
```

Comment:
Should this console log be wrapped if `if (IS_DEV) {}` or removed?

Reply:
removed